fedimint-api/src/task.rs

```diff
@@ -9,7 +9,10 @@ use futures::lock::Mutex;
 pub use imp::*;
 use thiserror::Error;
 #[cfg(not(target_family = "wasm"))]
+use tokio::sync::oneshot;
```

Comment:
Aside: My preference for rust module reference is growing into 'in-lined' use statements. In this instance, I'd have written
```
#[cfg(not(target_family = "wasm"))]
use tokio::{sync::oneshot, task::JoinHandle};
```
with the benefit that these two use references now share the wasm target family macro.
Do we have an implicit style preference for use statements?

Reply:
I tend to like them better too. We could enable them in `rustfmt.toml` (in another PR).